### PR TITLE
Add (stub-ish) research pages for accordion and disclosure.

### PR DESCRIPTION
**Pull request by dbaron**
This adds stub-ish research pages for the accordion and disclosure components.

### CODE REVIEW

**Comment by dbaron** [comment]
I think this (while still a stub) is now at least a working stub that could be reviewed and merged if folks think this sort of stub is appropriate.  The generated component pages are [Accordion](https://deploy-preview-721--open-ui.netlify.app/components/accordion.research/) and [Disclosure](https://deploy-preview-721--open-ui.netlify.app/components/disclosure.research/).

File: site/src/pages/components/accordion.research.mdx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+Some accordions are exclusive,
+which means that at any moment in time,
+only zero or one of the disclosure widgets can be open.
+In an exclusive accordion, opening one of the items
+closes any existing open item.
+(Of the design systems below,
+the accordion for Semantic UI is exclusive,
+the accordion for Stardust UI can be exclusive or not exclusive,
+and those for Carbon Design System, FAST, KoliBri, Lightning Design System,
+and Lion are not exclusive.)
```
Comment (mfreed7) [inline site/src/pages/components/accordion.research.mdx:21]: Does "not" mean "never" for the last list? If so, maybe s/not/never ?

**Review by mfreed7 — APPROVED** [review]
LGTM!

**Comment by mfreed7** [comment]
Alright, after figuring out how to add myself as a code owner, my approval above seems to be enough to land this. @dbaron feel free to do that when you want.

File: site/src/pages/components/accordion.research.mdx (hunk 1 above)
Comment (dbaron) [reply site/src/pages/components/accordion.research.mdx]: Done, and also added an "always" to the first list to match.

**Comment by mfreed7** [comment]
> Alright, after figuring out how to add myself as a code owner, my approval above seems to be enough to land this. @dbaron feel free to do that when you want.

Looks like my credentials are functional! 😄

**Pull request merged**